boot/bootutil/src/encrypted.c

```diff
@@ -79,15 +79,11 @@ key_unwrap(const uint8_t *wrapped, uint8_t *enckey)
 
     bootutil_aes_kw_init(&aes_kw);
     rc = bootutil_aes_kw_set_unwrap_key(&aes_kw, bootutil_enc_key.key, *bootutil_enc_key.len);
-    if (rc != 0) {
-        goto done;
-    }
-    rc = bootutil_aes_kw_unwrap(&aes_kw, wrapped, TLV_ENC_KW_SZ, enckey, BOOT_ENC_KEY_SIZE);
-    if (rc != 0) {
-        goto done;
+    
```

Comment:
I'm not sure I'm convinced this is an improvement. It makes the code a little more fragile if code ends up being added after this block, it could be missed how the error handling is done. The change wouldn't affect the generated code, regardless.

Reply:
Also, there is some spurious whitespace on the blank line here.